mod/01_rating/lib/stylesheets/style_drag_and_drop.scss

```diff
@@ -332,9 +332,15 @@ $bright-text-color: #e5e5e5;
       color:$dark-text-color;
 
     }
-    >.data {
-      width: auto!important;
+    >div{
+      >.data {
+        width: auto!important;
```

Comment:
'bang' you mean '!important' ? ( it is optional to have space before that, as many css minification tools removes the space )